### PR TITLE
Cleanup Wednesday #4

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Here we go again, this time with another round of server-side components that need not be anymore:

- remove legacy Jobs, Travel, Books, Masterclass and Live Event components
- simplify plista/outbrain feature switch
- remove some old CSS
- remove impossible ID in `advertsToInstantlyLoad`

Todo:
- [x] disable creative templates in DFP

### CODE REVIEW

File: static/src/javascripts-legacy/projects/commercial/modules/third-party-tags/plista.js [hunk 1]
```diff
@@ -81,7 +68,7 @@ define([
     };
 
     function init() {
-        if (shouldServe()) {
+        if (commercialFeatures.outbrain) {
```
Comment (rich-nguyen) [inline static/src/javascripts-legacy/projects/commercial/modules/third-party-tags/plista.js]: Does `commercialFeatures.outbrain` check for `isFront`?

**Review by rich-nguyen — APPROVED** [review]
ace 👍

File: static/src/javascripts-legacy/projects/commercial/modules/third-party-tags/plista.js (hunk 1 above)
Comment (rich-nguyen) [inline static/src/javascripts-legacy/projects/commercial/modules/third-party-tags/plista.js]: yes, through `isArticle`!

File: commercial/app/model/merchandise/jobs/Industries.scala [hunk 2]
```diff
@@ -6,8 +6,6 @@ import contentapi.ContentApiClient
 
 import scala.concurrent.Future
 
-case class JobSector(path: String, name: String)
-
 object Industries {
```
Comment (kelvin-chappell) [inline commercial/app/model/merchandise/jobs/Industries.scala]: Is this still used?

**Review by kelvin-chappell — APPROVED** [review]
Whoosh! 🚒

File: commercial/app/model/merchandise/jobs/Industries.scala (hunk 2 above)
Comment (regiskuckaertz) [inline commercial/app/model/merchandise/jobs/Industries.scala]: Yeah I think it is, in the `JobsAgent`.

**Comment by regiskuckaertz** [comment]
[Cleanup Wednesday Nb 4](https://trello.com/c/EUkiKGkP/225-cleanup-wednesday-nb-4)

**Pull request merged**